### PR TITLE
Add metric ignore support

### PR DESCRIPTION
**Pull request by henrydays**

### CODE REVIEW

File: configs.js [hunk 1]
```diff
@@ -222,6 +222,23 @@ function loadStandaloneConfig() {
 
         metrics = conf.metrics;
 
+        if (_.isArray(configs.ignore)) {
+            let filtered = metrics;
+
+            configs.ignore.forEach((metric) => {
+                // eslint-disable-next-line array-callback-return
+                filtered = filtered.filter(function (Metric) {
```
Comment (lfac-pt) [inline configs.js]: Why don't you use an arrow function here?

File: configs.js [hunk 2]
```diff
@@ -222,6 +222,23 @@ function loadStandaloneConfig() {
 
         metrics = conf.metrics;
 
+        if (_.isArray(configs.ignore)) {
+            let filtered = metrics;
```
Comment (lfac-pt) [inline configs.js:226]: Why don't we do something like:

```
metrics = metrics.filter((metric) => {
    return !configs.ignore.includes(current.info().name);
})
```

Comment (henrydays) [reply configs.js:226]: You cannot use info directly, must pass 3 args to the metric constructor

Comment (lfac-pt) [reply configs.js:226]: Why do we need to use the metric constructor?

File: configs.js (hunk 1 above)
Comment (henrydays) [reply configs.js]: done

File: configs.js (hunk 2 above)
Comment (henrydays) [reply configs.js:226]: yea, in order to access the info method

**Review by lfac-pt — APPROVED** [review]

**Pull request merged**